runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -25,6 +25,7 @@
 #include "chplrt.h"
 #include "chpl-env-gen.h"
 
+#include "chpl-atomics.h"
```

Comment:
What brought about this change in this PR? (Seems like we'd need atomics in ofi, just curious what about this PR brought it to your attention)